ecommerce/models_test.py

```diff
@@ -261,6 +261,16 @@ def test_product_version_save_text_id_badproduct(mocker):
     )
 
 
+def test_product_version_save_empty_description(mocker):
+    """ProductVersion should raise ValidationError if ProductVersion.description is empty"""
+    product_version = ProductVersionFactory.create(
+        product=ProductFactory.create(content_object=LineFactory())
+    )
+    product_version.description = ""
+    with pytest.raises(ValidationError):
```

Comment:
Can we make just a small addition here and check the validation error message as well.